projects/apache-poi/src/main/java/org/apache/poi/XLSX2CSVFuzzer.java

```diff
@@ -34,6 +34,13 @@ public static void fuzzerTestOneInput(byte[] input) {
 			OPCPackage p = OPCPackage.open(in);
 			XLSX2CSV xlsx2csv = new XLSX2CSV(p, NullPrintStream.INSTANCE, 5);
 			xlsx2csv.process();
+		} catch (UnsatisfiedLinkError e) {
+			// only allow one missing library related to Font/Color-handling
+			// we cannot install additional libraries in oss-fuzz images currently
+			// see https://github.com/google/oss-fuzz/issues/7380
+			if (!e.getMessage().contains("libawt_xawt.so")) {
```

Comment:
I see the issue here https://github.com/google/oss-fuzz/issues/7380 but what is the different between "installing" and "bundling" the libraries? You can add the relevant dynamic libraries to the runtime, see e.g. https://google.github.io/oss-fuzz/further-reading/fuzzer-environment/#runtime-dependencies 

Doesn't that qualify as "installing" a package in this case? If not, what's the difference?

Reply:
Ah, I think you did go over this in the issue actually, I just saw the [comment](https://github.com/google/oss-fuzz/issues/7380#issuecomment-1074789681) `the font-system is more than just some so-libs and the fonts, it includes some config-files and other pieces, not sure if it can be packaged along the application.`.

Reply:
Yes, unfortunately it's more complex than simply bundling some .so-files. 

However Java fails quite easily when fonts are missing, e.g. using `java.awt.Color` and other stuff from java.awt triggers this exception in oss-fuzz, so finding some workaround would be nice to avoid bigger blind spots because of this.

Reply:
Do you know/have an estimate fro how much more is it than just moving .so files?

Reply:
I did a closer look and it seems that most of it can work if we just bundle a few libraries, I will try to do a PR if local testing doesn't uncover any other problem.

Reply:
We now put some .so files in place via #11006 which should avoid such exceptions and allow to cover more code in Apache POI via fuzzing.